manage/upgrading/version_specific_migration/upgrade_to_python3.rst

```diff
@@ -209,74 +217,77 @@ Precompile will be run every time you run buildout. If you want to avoid running
 
-This kind of error message
+This kind of error message:
 
 .. code-block:: shell
 
-   TypeError: Class advice impossible in Python3.  Use the @implementer class decorator instead.
+    TypeError: Class advice impossible in Python3.  Use the @implementer class decorator instead.
 
 tells you that there is a class that is using an ``implements`` statement which needs to be replaced by the ``@implementer`` decorator.
 
-Example, this kind of code:
+For example, this kind of code:
 
 .. code-block:: python
 
-   from zope.interface import implements
+    from zope.interface import implements
 
-   class Group(form.BaseForm):
-       implements(interface.IGroup)
-       …
+    class Group(form.BaseForm):
+        implements(interface.IGroup)
 
 needs to be replaced with:
 
 .. code-block:: python
 
-   from zope.interface import implementer
+    from zope.interface import implementer
 
-   @implementer(interfaces.IGroup)
-   class Group(form.BaseForm):
-       …
+    @implementer(interfaces.IGroup)
+    class Group(form.BaseForm):
 
-The same is the case for `provides(IFoo)` and some other Class advices. These need to be replaced with their respective decorators like `@provider`.
+The same is the case for ``provides(IFoo)`` and some other Class advices.
+These need to be replaced with their respective decorators like ``@provider``.
 
 
 B - Relative Imports
 ^^^^^^^^^^^^^^^^^^^^
 
-Relative imports like `import permissions` are no longer permitted. Use `from collective.package import permissions` or `from . import permissions` (not recommended)
+Relative imports like ``import permissions`` are no longer permitted.
+Use ``from collective.package import permissions`` or ``from . import permissions`` (not recommended).
 
 
 C - Syntax Error On Importing Async
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-In Python 3.7 you can no longer have a module called `async` (see https://github.com/celery/celery/issues/4849). You need to rename all such files, folders or packages (like zc.async and plone.app.async).
+In Python 3.7 you can no longer have a module called ``async`` (see https://github.com/celery/celery/issues/4849).
+You need to rename all such files, folders or packages (like ``zc.async`` and ``plone.app.async``).
 
 
 5 Test functionality manually
 -----------------------------
 
-Now that the instance is running you should do the following and fix all errors as the appear.
+Now that the instance is running you should do the following and fix all errors as they appear.
 
-* Install the addon
-* Test basic functionality (e.g. adding and editing content-types and views)
-* Uninstall the addon
+* Install the add-on.
+* Test basic functionality (e.g., adding and editing content-types and views).
+* Uninstall the add-on.
 
-For this step you should have ``Products.PDBDebugMode`` installed. It will make fixing any issues much easier.
+For this step you should have ``Products.PDBDebugMode`` installed.
+It will make fixing any issues much easier.
```

Comment:
Personally, I would not use "much easier" this can be misleading.

Nothing is 'easy' what is 'easy' for you may not 'easy' for others.
You could change that to maybe effortless or something else?

Reply:
Reworded in 4d5ab52